selfdrive/car/volkswagen/carstate.py

```diff
@@ -41,9 +42,10 @@ def update(self, pt_cp, cam_cp, ext_cp, trans_type):
     ret.yawRate = pt_cp.vl["ESP_02"]["ESP_Gierrate"] * (1, -1)[int(pt_cp.vl["ESP_02"]["ESP_VZ_Gierrate"])] * CV.DEG_TO_RAD
 
     # Verify EPS readiness to accept steering commands
+    # For cars without Lane Assist, treat FAULT like INITIALIZING for worst likely controls init and EPS recovery time
     hca_status = self.hca_status_values.get(pt_cp.vl["LH_EPS_03"]["EPS_HCA_Status"])
-    ret.steerFaultPermanent = hca_status in ("DISABLED", "FAULT")
-    ret.steerFaultTemporary = hca_status in ("INITIALIZING", "REJECTED")
+    ret.steerFaultPermanent = hca_status == "DISABLED" or (hca_status == "FAULT" and self.frame >= 600)
+    ret.steerFaultTemporary = hca_status in ("INITIALIZING", "REJECTED") or (hca_status == "FAULT" and self.frame < 600)
```

Comment:
can you also check the falling edge? I can already see this popup in the MTBF report where a user engages in the first 6s with the EPS in the FAULT state